reV/supply_curve/supply_curve.py

```diff
@@ -735,6 +734,18 @@ def compute_total_lcoe(self, fcr, transmission_costs=None,
             cost /= self._trans_table['capacity']  # $/MW
             self._trans_table['trans_cap_cost_per_mw'] = cost
 
+        if 'reinforcement_cost' in self._trans_table:
```

Comment:
maybe add an info logging statement if this is found? I always find that i put these conditionals in here, then it doesn't work, and i spend a ton of time debugging when i just made a typo in the string or the table header. It's also nice to know that this bit of code is running.

Reply:
Good point! Added below